packages/ripple-binary-codec/src/binary.ts

```diff
@@ -66,8 +66,9 @@ function serializeObject(object: JsonObject, opts: OptionObject = {}): Buffer {
   const filter = signingFieldsOnly
     ? (f: FieldInstance): boolean => f.isSigningField
     : undefined
-
-  coreTypes.STObject.from(object, filter).toBytesSink(bytesList)
+  ;(coreTypes.STObject as typeof STObject)
```

Comment:
What does the `;` do here?

Reply:
The `;` is used to prevent the interpreter from treating the casted object as a parameter of an `undefined` function like so: (This is what the code becomes with prettier otherwise - it can't tell automatically where the variable declaration ends if it's a statement I believe)
```
  const filter = signingFieldsOnly
    ? (f: FieldInstance): boolean => f.isSigningField
    : undefined(coreTypes.STObject as typeof STObject)
```